src/containerapp/azext_containerapp/custom.py

```diff
@@ -2629,3 +2632,122 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+
+def create_session_pool(cmd,
+                        name,
+                        resource_group_name,
+                        location=None,
+                        managed_env=None,
+                        container_type=None,
+                        cooldown_period=None,
+                        secrets=None,
+                        egress_enabled=None,
+                        max_concurrent_sessions=None,
+                        ready_session_instances=None,
+                        image=None,
+                        container_name=None,
+                        cpu=None,
+                        memory=None,
+                        env_vars=None,
+                        startup_command=None,
+                        args=None,
+                        target_port=None,
+                        registry_server=None,
+                        registry_pass=None,
+                        registry_user=None):
+    raw_parameters = locals()
+    session_pool_decorator = SessionPoolCreateDecorator(
+        cmd=cmd,
+        client=SessionPoolPreviewClient,
+        raw_parameters=raw_parameters,
+        models=SessionPoolModel
```

Comment:
The `model` is only for specify the dir of `sdk_models`, you should use `models=CONTAINER_APPS_SDK_MODELS`